check_unattended_upgrades.py

```diff
@@ -175,6 +175,7 @@ def get_argparser() -> argparse.ArgumentParser:
         "--repo",
         "--custom-repo",
         dest="custom_repo",
+        action='append',
         help="Check if 'Unattended-upgrades' is configured to include the "
```

Comment:
Add a note to the help message that the --repo option can be applied multiple times.